src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="borders">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--5">
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
+        <div class="stars__star stars__star--active"></div>
```

Comment:
stars__star--active modifiers are redundant.
use stars--1, stars--2 ... stars--5 modifiers for show active stars